css/statusPage.css

```diff
@@ -0,0 +1,104 @@
+.status-page-container {
+    margin: 4em auto;
+    max-width: 1000px;
+    width: 90vw;
+    height: 90vh;
+    background-color: white;
+    color: black;
+    z-index: 100000;
+    position: absolute;
+    box-sizing: border-box;
+    padding: 2em;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    overflow-y: scroll;
+    font-size: 0.75em;
+}
+
+.status-page-header {
+    margin-bottom: 1em;
+    display: flex;
+    flex-direction: row;
+    gap: 1em;
+    align-items: center;
+}
+
+.status-page-title {
+    font-size: 3em;
+    font-weight: bold;
+}
+
+.status-page-button {
+    border: 1px solid black;
+    border-radius: 2px;
+    padding: 4px 8px;
+    background-color: #F0F0F0;
+    display: inline-block;
+    cursor: pointer;
+    
+    &:hover {
```

Comment:
Somehow I didn't know you can nest styles with `&`, this is good to know 😆

Reply:
It's fully supported in browsers as of end of 2023 actually https://developer.mozilla.org/en-US/docs/Web/CSS/Nesting_selector